### ISSUE TITLE
Can't seem to find '_debugger' from DefinitelyTyped

### ISSUE THREAD

**Issue by DanielRosenwasser** [DefinitelyTyped/tsd#193]
[`_debugger.d.ts`](https://github.com/borisyankov/DefinitelyTyped/blob/92ad4a9922ff061ad208eb840bfe66d072d32de4/_debugger/_debugger.d.ts) seems to exist on DefinitelyTyped.

```
$ tsd install _debugger

>> zero results

$ tsd query _debugger

>> zero results
```

Similarly, if I do `tsd query *` I can't find '_debugger'.

**Comment by DanielRosenwasser** [comment]
Additionally, if I try adding

```JSON
"_debugger/_debugger.d.ts": {
  "commit": "92ad4a9922ff061ad208eb840bfe66d072d32de4"
}
```

to my `tsd.json` along with a `tsd update`, and got the following:

```
[1] _debugger&#x2f;_debugger.d.ts is not allowed
    at root.assert (C:\Users\drosen\AppData\Roaming\npm\node_modules\tsd\node_modules\joi\lib\index.js:121:19)
    at Config.validateJSON (C:\Users\drosen\AppData\Roaming\npm\node_modules\tsd\build\tsd\context\Config.js:158:13)
    at Config.parseJSON (C:\Users\drosen\AppData\Roaming\npm\node_modules\tsd\build\tsd\context\Config.js:131:14)
    at Config.parseJSONString (C:\Users\drosen\AppData\Roaming\npm\node_modules\tsd\build\tsd\context\Config.js:125:14)
    at C:\Users\drosen\AppData\Roaming\npm\node_modules\tsd\build\tsd\logic\ConfigIO.js:44:43
    at tryCatch1 (C:\Users\drosen\AppData\Roaming\npm\node_modules\tsd\node_modules\bluebird\js\main\util.js:63:19)
    at Promise$_callHandler [as _callHandler] (C:\Users\drosen\AppData\Roaming\npm\node_modules\tsd\node_modules\bluebird\js\main\promise.js:695:13)
    at Promise$_settlePromiseFromHandler [as _settlePromiseFromHandler] (C:\Users\drosen\AppData\Roaming\npm\node_modules\tsd\node_modules\bluebird\js\main\promise.js:711:18)
    at Promise$_settlePromiseAt [as _settlePromiseAt] (C:\Users\drosen\AppData\Roaming\npm\node_modules\tsd\node_modules\bluebird\js\main\promise.js:868:14)
    at Promise$_settlePromises [as _settlePromises] (C:\Users\drosen\AppData\Roaming\npm\node_modules\tsd\node_modules\bluebird\js\main\promise.js:1006:14)
```

I tried fixing the relevant regular expression, but somehow certain tests were failing on me with or without my changes.

**Comment by Diullei** [comment]
Hi @DanielRosenwasser! The tests fail when run on windows. On Linux everything works fine. I'll fix this test problem first and then check this issue. Thank you for reporting!

**Comment by Diullei** [comment]
Hi!

fixed with `0.6.5-beta.2`

```
npm install tsd@0.6.5-beta.2
```

https://github.com/DefinitelyTyped/tsd/releases/tag/0.6.5-beta.2

**Comment by DanielRosenwasser** [comment]
Thanks for the fix @Diullei!
